### DISCUSSION TITLE
Firebase Cloud Messaging in CSB

### DISCUSSION THREAD

**Discussion by weilies** [codesandbox/codesandbox-client#4071] [General] [answered]
Greeting gurus,

i  need to make a 'Show Notification' => Allow/Block effect using Firebase Cloud Messaging
here my project: https://codesandbox.io/s/csb-messaging-x-lzem4
here my sample site: https://lzem4.csb.app/
sample config from [youtube](https://www.youtube.com/watch?v=gV3RbBgjEEU&lc=Ugys2dYXksYb_EevGF14AaABAg.988xXWxVKCC98BKuzuseMU)

But i am not sure why i getting tons of error in console
```
Manifest: property 'scope' ignored. Start url should be within scope of scope URL
DevTools failed to load SourceMap: Could not parse content for https://lzem4.csb.app/node_modules/@angular/core/bundles/core.umd.js.map: Unexpected token < in JSON at position 0
DevTools failed to load SourceMap: Could not parse content for https://lzem4.csb.app/node_modules/rxjs/index.js.map: Unexpected token < in JSON at position 0
...
```
appreciated some helps here. 

Env:

- Browser - Chrome
- OS - Chromebook v81

Thanks

**Comment by garethx** [comment, marked as answer]
They're not errors, just warnings and most don't relate specifically to your code but how sourcemaps are being handled within CodeSandbox itself. They're ok to ignore.

**Reply by weilies** [reply]
thanks @garethx ! but looks like the problem is with Firebase. Please share with me if you happened to know a working example to work with Firebase Cloud messaging :)
